### ISSUE TITLE
Set ring buffers to hardware max on network-{throughput,latency}

### ISSUE THREAD

**Issue by jcpunk** [redhat-performance/tuned#660]
https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/monitoring_and_managing_system_status_and_performance/tuning-the-network-performance_monitoring-and-managing-system-status-and-performance#increasing-the-ring-buffers-to-reduce-a-high-packet-drop-rate_tuning-network-adapter-settings

It would be helpful if the network profiles were able to set the hardware ring buffers to match the hardware max of the network cards.

**Comment by techiesaurus** [comment]
This makes perfect sense to me for throughput for receive packet buffers, throughput & max. transmit buffers not always good idea--in particular when still seeing some receive-side drops after max. receive buffer applied.  
Some latency-sensitive systems are short distance and loss tolerant--this is not what those systems need.

**Comment by sprat** [comment]
Yes, we also would like to be able to set the value to the max. But for the wording, instead of `auto` we can use `max` which would be more explicit.
